.gitignore

```diff
@@ -6,3 +6,5 @@ __pycache__
 .idea/
 .coverage*
 .mypy_cache
+notebooks/.DS_Store
+.DS_Store
```

Comment:
Please move .gitignore modifications to a separate PR.

Reply:
This was mostly here for my own sanity. Unfortunately I either have to add them to .gitignore or the files will show up in the PR because they change every time you so much as breathe on a Mac. I've noticed that neither you nor Lon have them in .gitignore though -- how do you get around .DS_Store showing up everywhere?